.github/workflows/main.yaml

```diff
@@ -356,18 +352,28 @@ jobs:
           release_name: Release ${{ needs.setup.outputs.VERSION }}
           draft: true
           prerelease: false
-      - name: Upload Release Asset Windows
-        id: upload-release-asset-windows
+
+      - name: Upload Release Asset Windows Installer
         uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }} # This pulls from the CREATE RELEASE step above, referencing it's ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps 
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
           asset_path: artifacts/wasmer-windows-amd64/WasmerInstaller.exe
-          asset_name: wasmer-windows.exe
+          asset_name: wasmer-windows-installer.exe
```

Comment:
This may break `wasmer-install`: https://github.com/wasmerio/wasmer-install/blob/master/install.ps1#L42

Reply:
Yup we will need to update it prior the next release

Reply:
Agree. Even if we update `install.ps1`, we must keep backward compatibility. I'll revert that commit.